prov/rxm/src/rxm_init.c

```diff
@@ -378,6 +378,10 @@ RXM_INI
 			"(default: 128). Setting this to 0 would get default "
 			"value defined by the MSG provider.");
 
+	fi_param_define(&rxm_prov, "conn_quota", FI_PARAM_SIZE_T,
+			"Defines how many connections per RxM EP can "
+			"progress simultaneously (default: 64).");
+
```

Comment:
Why do we need this knob? Please add the reason to commit message.

Reply:
Described in the commit message